crates/uv/src/commands/build_frontend.rs

```diff
@@ -187,14 +187,19 @@ async fn build_impl(
             }
         };
 
-        let project = workspace
+        let package = workspace
             .packages()
             .get(package)
-            .ok_or_else(|| anyhow::anyhow!("Package `{}` not found in workspace", package))?
-            .root();
+            .ok_or_else(|| anyhow::anyhow!("Package `{package}` not found in workspace"))?;
+
+        if !package.pyproject_toml().is_package() {
+            let name = &package.project().name;
+            let pyproject_toml = package.root().join("pyproject.toml");
+            return Err(anyhow::anyhow!("Package `{}` is missing a `{}`. For example, to build with `{}`, add the following to `{}`:\n```toml\n[build-system]\nrequires = [\"setuptools\"]\nbuild-backend = \"setuptools.build_meta\"\n```", name.cyan(), "build-system".green(), "setuptools".cyan(), pyproject_toml.user_display().cyan()));
```

Comment:
Should I syntax-highlight the code...?

Reply:
I'd offset the part of the code you need to copy.

Could you wrap the line? I like indoc for multiline strings.

Should we be recommending setuptools here when we're using hatchling in uv init?